src/pages/ResearcherProfile.js

```diff
@@ -465,7 +446,28 @@ export const ResearcherProfile = hh(class ResearcherProfile extends Component {
     return _.omit(_.cloneDeep(profile), ['libraryCards', 'libraryCardEntries']);
   };
 
+  generateCountryNames = () => {
```

Comment:
Good call with the helper functions, keeps the variables contained and sharable when needed.